src/log/service.ts

```diff
@@ -67,3 +71,11 @@ function isContext(ctx: any): ctx is Context {
   return ((<Context> ctx).ip !== undefined);
 
 }
+
+function getCountryByIp(ip: string)
```

Comment:
I had a look in this library, and I think it's worth making a small adjustment.

The library will return a country, _or_ the string `?`. I don't think it's good to store `?` anywhere, so instead I think this function should have a condition for this and return `null` if a country could not be determined.